test/integration/hotrestart_test.sh

```diff
@@ -16,11 +16,19 @@ cat "${TEST_RUNDIR}"/test/config/integration/server.json |
 # Now start the real server, hot restart it twice, and shut it all down as a basic hot restart
 # sanity test.
 echo "Starting epoch 0"
+ADMIN_ADDRESS_PATH_0="${TEST_TMPDIR}"/admin_0.address
 "${ENVOY_BIN}" -c "${HOT_RESTART_JSON}" \
-    --restart-epoch 0 --base-id 1 --service-cluster cluster --service-node node &
+    --restart-epoch 0 --base-id 1 --service-cluster cluster --service-node node \
+    -a "${ADMIN_ADDRESS_PATH_0}" &
 
 FIRST_SERVER_PID=$!
 sleep 3
+
+echo "Updating original config json listener addresses"
+UPDATED_HOT_RESTART_JSON="${TEST_TMPDIR}"/hot_restart_updated.json
+tools/socket_passing.py "-o" "${HOT_RESTART_JSON}" "-a" "${ADMIN_ADDRESS_PATH_0}" \
```

Comment:
This should be `${TEST_RUNDIR}"/tools/socket_passing.py` since #819 (sorry about the churn!).